src/escalus_session.erl

```diff
@@ -121,9 +121,9 @@ session(Conn, Props) ->
     Props.
 
 use_ssl(Props, Features) ->
-    UserNeedSSL = proplists:get_value(ssl, Props, false),
-    StreamAllowSSL = proplists:get_value(starttls, Features),
-    case {UserNeedSSL, StreamAllowSSL} of
+    UserNeedsSSL = proplists:get_value(starttls, Props, false),
```

Comment:
This changes public interface (the configuration file) of the library in a backwards incompatible way (by adding `starttls` and redefining `ssl`). I would have loved it to be reflected in `priv/escalus.config` before bisecting the history in search of this tiny little bit ;)